src/main/java/io/javalin/core/JavalinConfig.java

```diff
@@ -134,9 +175,25 @@ public void server(Supplier<Server> server) {
 
     public static void applyUserConfig(Javalin app, JavalinConfig config, Consumer<JavalinConfig> userConfig) {
         userConfig.accept(config); // apply user config to the default config
+
+        // The following handlers need to be initialized before any handler is added
+        RouteOverviewRenderer routeOverviewRenderer = null;
         if (config.inner.routeOverview != null) {
-            app.get(config.inner.routeOverview.getPath(), new RouteOverviewRenderer(app), config.inner.routeOverview.getRoles());
+            routeOverviewRenderer = new RouteOverviewRenderer(app);
+        }
+        OpenApiHandler openApiHandler = null;
+        if (config.inner.openApiOptions != null) {
+            openApiHandler = new OpenApiHandler(app, config.inner.openApiOptions);
+            config.inner.openApiHandler = openApiHandler;
+        }
+
```

Comment:
Make api docs available here ?

Reply:
What do you mean?

Reply:
To create an endpoint which will expose the docs using swagger ui.

Reply:
> I plan to provide the schema as in a swagger ui directly, but I wanted to do this as a separate PR, as this one is already big enough. Upgrading the version would be a major change.

I think this answer the question :) I will do this in a separate PR